Digipost.Signature.Api.Client.Direct.Tests/DataTransferObjects/DataTransferObjectConverterTests.cs

```diff
@@ -173,79 +173,6 @@ public void Converts_direct_job_status_with_multiple_signers_successfully()
                 Assert.Empty(differences);
             }
 
-            [Fact]
-            public void Converts_direct_job_successfully()
-            {
```

Comment:
Vi bestemte oss for en tid tilbake å flytte ansvaret for konvertering fra deserialiserte klasser til domeneobjekter til objektet selv. Vi gjør det derfor gradvis, og denne gangen ble det responsen, siden det er den vi endrer på i denne omgangen. Denne er altså bare flyttet.